org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host"
+    ],
+    "build-options" : {
+	"build-args": [ "--share=network" ],
```

Comment:
You can't have network access at build time as the builders are sandboxed.

Reply:
oh okay